test/signalrclienttests/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set (SOURCES
   test_websocket_client.cpp
   url_builder_tests.cpp
   websocket_transport_tests.cpp
+  signalr_default_scheduler_tests.cpp
```

Comment:
Do we have tests that verify you can swap the scheduler for your own implementation?

Reply:
Added, a little ugly because it hardcodes an expected schedule amount. But there isn't any other easy way to check that it worked, and hopefully that number will be stable.